Mathlib/Algebra/Order/AbsoluteValue.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Anne Baanen
 
 ! This file was ported from Lean 3 source module algebra.order.absolute_value
-! leanprover-community/mathlib commit fc2ed6f838ce7c9b7c7171e58d78eaf7b438fb0e
+! leanprover-community/mathlib commit 7ea604785a41a0681eac70c5a82372493dbefc68
```

Comment:
https://leanprover-community.github.io/mathlib-port-status/file/algebra/order/absolute_value?range=fc2ed6f838ce7c9b7c7171e58d78eaf7b438fb0e..7ea604785a41a0681eac70c5a82372493dbefc68

Reply:
I'm confused; this says that the only change to this file is a single instance, so where did the other changes come from? Was the sychronization comment wrong?

Reply:
I think so, yes.

Reply:
I think so, yes.

Reply:
Do you know what the right value is? If so, a PR that just corrects it would make further review easier.